src/test/java/nextstep/ladder/LineTest.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.ladder;
+
+import nextstep.ladder.domain.Line;
+import nextstep.ladder.domain.Point;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LineTest {
+
+    @Test
+    @DisplayName("라인이 겹칠 때 생성 불가 테스트")
+    void lineOverLabTest() {
+        assertThatThrownBy(() -> new Line(List.of(new Point(true), new Point(true), new Point(false))))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("중복");
+    }
+
+    @Test
+    @DisplayName("라인 생성 테스트")
+    void lineConstructTest() {
+        Line line = new Line(List.of(new Point(false),new Point(true),new Point(false)));
+        assertThat(line.pointToBoolean().get(1)).isEqualTo(true);
```

Comment:
값을 꺼내야하나요?
정말 궁금한것은 무엇인가요?

```suggestion
        assertThat(line.pointToBoolean()).containsExactly(...);
```

객체를 단순한 자료구조로 바라보기 보다는 어떠한 상태를 주체적으로 관리하는 객체로 생각해보아요.

```suggestion
        assertThat(line.findPoint(1)).isEqualTo(true);
```

`pointToBoolean` 가 궁금한것인가요, 위치에 따른 상태가 궁금한것인가요?

Reply:
해당 테스트는 라인이 제대로 생성됐는지 확인하려는 목적입니다.
`containsExactly` 사용이 맞는 것 같습니다!